cmd/agent/app/options/options.go

```diff
@@ -164,10 +173,14 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 		"A list of controllers to enable. '*' enables all on-by-default controllers, 'foo' enables the controller named 'foo', '-foo' disables the controller named 'foo'. All controllers: %s.",
 		strings.Join(allControllers, ", "),
 	))
-	fs.StringVar(&o.BindAddress, "bind-address", defaultBindAddress,
+	fs.StringVar(&o.BindAddress, "bind-address", o.BindAddress,
 		"The IP address on which to listen for the --secure-port port.")
-	fs.IntVar(&o.SecurePort, "secure-port", defaultPort,
+	fs.IntVar(&o.SecurePort, "secure-port", o.SecurePort,
```

Comment:
We don't need to change the default value here. As the old default will be transmit to new default.

Reply:
Would it be better to put the default value in usage as well as `o.HealthProbeBindAddress`?  So that we can figure out if the user still use the old flag in `complete` phase.